src/components/PaginatedTable/PaginatedTable.css

```diff
@@ -0,0 +1,47 @@
+@import '../../styles/theme/index.css';
+:root {
+  --table-footer-background-color: var(--structure-color-200);
+  --table-font-family: 'Inter';
+  --table-header-color: var(--structure-color-900);
+  --table-font-family: 'Inter';
+  --table-footer-font-weight: 300;
+  --table-padding: 10px;
+  --table-font-size: 14px;
+}
+
+tfoot td.paginated-table-action-btns {
```

Comment:
@branquinhoaa please take care to not rely of HTML references for CSS references. We should always use a custom class reference. This may produce side-effects.